llama-index-integrations/tools/llama-index-tools-azure-dynamic-sessions/README.md

```diff
@@ -0,0 +1,45 @@
+# Azure Dynamic Sessions Tool

```

Comment:
I know this is the name of the service, but we could also consider a more direct name if we wanted (azure-code-execution, azure-code-sandbox, etc.). Up to you!

Reply:
Let me double check on this one. I believe we eventually settled with the name Dynamic Sessions. I'll verify that just in case

Reply:
Since this package for LlamaIndex is specific to the code interpreter tool, it makes sense to make the name specific too.

I think `llama-index-tools-azure-sessions-code-interpreter` could work. Perhaps even shorten it to `llama-index-tools-azure-code-interpreter`.

Reply:
ok with keeping it `llama-index-tools-azure-dynamic-sessions` too though

Reply:
I went with `llama-index-tools-azure-code-interpreter`, but open to changing again!